app/cdap/components/GridTable/components/AaToolbar/images.tsx

```diff
@@ -0,0 +1,242 @@
+/*
```

Comment:
modify name of this component as TransformationsToolbar

Reply:
done